mtp_api/templates/performance/digital-takeup-report.html

```diff
@@ -9,16 +9,13 @@
   {{ block.super }}
 
   <link rel="stylesheet" type="text/css" href="{% static 'admin/css/changelists.css' %}" />
+  <link rel="stylesheet" type="text/css" href="{% static 'stylesheets/digital-takeup-reports.css' %}"/>
   <style>
     #takeup-chart {
```

Comment:
Is it worth the other css rules to the stylesheet?
Also- I couldn't see the file linked by `{% static 'admin/css/changelists.css' %}` anywhere, where does this get pulled in from?

Reply:
that style sheet come from Django itself. it's one of the many that are automatically used in various parts of the admin interface.